site/src/uiuisms.rs

```diff
@@ -228,15 +228,15 @@ uiuisms!(
     /// Dot product
     "/+× [1 2 3] [4 ¯5 6]",
     /// Cross product
-    "1_2_3 4_5_6\n↻1-∩(×↻1)⊃:⊙∘",
+    "1_2_3 4_5_6\n↻2-∩(×↻2)⊙.,",
```

Comment:
I think I prefer `⊃:⊙∘`

Reply:
I think my most preferred way of writing it would be `:,,` since `,,` can be recognized as "dupe the top 2 stack items," but the style checker complains about `:,`.

Reply:
You can use `:,,`. I'll fix the lint.